### DISCUSSION TITLE
an issue about lagging

### DISCUSSION THREAD

**Discussion by alperengulen** [ppy/osu#26646] [Q&A]
### Type

Performance

### Bug description

while playing it feels like mouse sensivity is getting decerased by 5-10 sometimes and one time cursor slowed down and got faster for a moment like the thing that happens in online games when you have ping (sorry for the bad english)

### Version

2024.114.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13999424/compressed-logs.zip)

**Comment by peppy** [comment]
Sounds like https://github.com/ppy/osu-framework/issues/6141.
